### DISCUSSION TITLE
How to uninstall Explorer Patcher

### DISCUSSION THREAD

**Discussion by FlameyJamey** [valinet/ExplorerPatcher#1345] [Q&A] [answered]
Hello, I was wondering if anyone can help me by giving me instructions on how to uninstall Explorer Patcher, As i have decided that it is just annoying to have installed!
Thanks in advance!
:)

**Comment by tallman69** [comment, marked as answer]
You can use one of these three methods to uninstall ExplorerPatcher:

-Use "Programs and Features" in Control Panel, or "Apps and features" in the Settings app or
-Run **ep_setup.exe /uninstall** or
-Rename **ep_setup.exe** to **ep_uninstall.exe** and run that.

**Reply by FlameyJamey** [reply]
Where can I find those files?

**Reply by FlameyJamey** [reply]
Also, It doesn't show up in the programs and features settings in windows? (I'm using windows 11 btw)

**Reply by abdumalik68** [reply]
You have to rename file dxgi.dll to dxgio.dll in C:/Windows and delete folder Explorer Patcher in %appdata% then restart PC

**Reply by topia** [reply]
although you can safely rename dxgi.dll in `C:\Windows` directory, ExplorerPatcher installer modifies somewhere else, so running uninstaller (for example, `"%ProgramFiles%\ExplorerPatcher\ep_setup.exe" /uninstall`) is also necessary.

**Reply by FlameyJamey** [reply]
Thanks for the help! I have uninstalled it and all I needed to do was install ep_setup.exe and open programs and features!

**Reply by tallman69** [reply]
Good!  Mission accomplished!

**Reply by Asad-13** [reply]
I had once renamed ep_setup.exe to explorerpatcher_setup.exe for my convenience. Now I renamed it to ep_uninstall.exe and ran it. The uninstall dialogue came up fine, but my Explorer hasn't restarted, and bringing up Task Manager shows that the uninstaller is still running. What can I do?

**Reply by pyrates999** [reply]
reboot

**Reply by palagar** [reply]
agora o windows 11 atualizou e adicionou a opção de não combinar.

**Comment by impeeza** [comment]
Hello  there after some time my company prohibited the execution of non authorized program, and gues what, explorer patcher is not withe listed and will not be. so even if I go to safe mode can not run ep_setup.exe nor ep_uninstall.exe, so no way to automatically remove explorer patcher, how can in safe mode remove explorer patcher manually?

Thanks.

**Reply by tallman69** [reply]
See this tutorial.  https://www.tenforums.com/tutorials/143263-how-install-uninstall-programs-safe-mode-windows-10-a.html

**Reply by impeeza** [reply]
will try, for now renaming dxgi.dll let me to start explorer, trying to get the block from central adminstrator. Thanks a lot.

**Reply by Anayaali2019** [reply]
after uninstall i am facing some issue like start menu not opening  also explorer window popup randomly

**Reply by pyrates999** [reply]
Please do not hijack a thread.  Please create a new post in discussions for the issue you are facing.

**Comment by yougs21** [comment]
Hello,
how to do a fully silent uninstallation with version 22621.1992.56.3 ?
I tried renaming ep_setup to ep_uninstall but it still asks me to validate, I can't script that. 
Thanks

**Reply by pyrates999** [reply]
you need to start the batch file that runs ep_uninstall.exe to start with elevated privileges.  The prompt though may be because it is not signed.

**Reply by yougs21** [reply]
Hello, the script is launched with the rights admitted, I was not talking about the rights elevation window but a request "Are you sure you want to remove EP from your computer".
I tried several arguments but without success

**Reply by pyrates999** [reply]
Ok, so a quiet uninstall is what you want.

**Reply by pyrates999** [reply]
Official request for this is here: https://github.com/valinet/ExplorerPatcher/discussions/2007

**Comment by JoeStipher** [comment]
After I uninstalled explorer patcher, I can't open any files at all in the file explorer. How can I solve this issue?

**Reply by pyrates999** [reply]
It's probably the applications themselves that are crashing.

1. go to Event Viewer and go to Windows Logs -> Application
2. if you see an entry like this:
```
Fault bucket 2160498250610915749, type 5
Event Name: BEX
Response: Not available
Cab Id: 0

Problem signature:
P1: App.exe
P2: 8.101.0.212
P3: 62d753ca
P4: StackHash_1abe
P5: 0.0.0.0
P6: 00000000
P7: PCH_0C_FROM_ntdll+0x0007467C
P8: c0000409
P9: 0000000a
P10: 

Attached files:
\\?\C:\ProgramData\Microsoft\Windows\WER\Temp\WERE674.tmp.dmp
\\?\C:\ProgramData\Microsoft\Windows\WER\Temp\WERE750.tmp.WERInternalMetadata.xml
\\?\C:\ProgramData\Microsoft\Windows\WER\Temp\WERE761.tmp.xml
\\?\C:\ProgramData\Microsoft\Windows\WER\Temp\WERE765.tmp.csv
\\?\C:\ProgramData\Microsoft\Windows\WER\Temp\WERE775.tmp.txt

These files may be available here:
\\?\C:\ProgramData\Microsoft\Windows\WER\ReportArchive\AppCrash_App.exe_4d99db35ae3a95a5a9fe01cdd8f6f963aae2af2_b0ea042c_15cf544f-9943-4501-8e58-1f41c519d6a0

Analysis symbol: 
Rechecking for solution: 0
Report Id: 458c1bcc-2bc8-42b3-88b8-8b390fa6f27b
Report Status: 268435456
Hashed bucket: 411b6225a47294726dfba1b64d8439a5
Cab Guid: 0
```
3. Reboot and you should be able to open apps again

Another way is to open up task manager and kill explorer.exe and then run task manager again and run explorer.exe
